<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>318</width>
    <height>400</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>318</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MULYANKAN</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>10</y>
      <width>304</width>
      <height>330</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>300</width>
      <height>300</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>400</width>
      <height>330</height>
     </size>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="result_eval_tab">
     <attribute name="title">
      <string>Result Evaluator</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Garuda</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Paper Code :</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QComboBox" name="paper_code_list">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>70</width>
          <height>30</height>
         </size>
        </property>
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="minimumContentsLength">
         <number>1</number>
        </property>
        <property name="iconSize">
         <size>
          <width>8</width>
          <height>16</height>
         </size>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <item>
         <property name="text">
          <string>121</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>122</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>123</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manage</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QPushButton" name="response_select">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLabel" name="response_path">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="key_select">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLabel" name="key_path">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QPushButton" name="result_eval_submit">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>30</height>
         </size>
        </property>
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>Garuda</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Insert Key File (.txt) :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Garuda</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Insert Response File (.txt) :</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="center_alloc_tab">
     <attribute name="title">
      <string>Center Allocator</string>
     </attribute>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="4" column="0">
       <widget class="QPushButton" name="student_info_select">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QPushButton" name="center_infor_select">
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLabel" name="center_info_path">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="3">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Garuda</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Insert Center Info. File (.xls) :</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="3">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Garuda</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <italic>true</italic>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Insert Student Info File (.xls :)</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QLabel" name="student_info_path">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QPushButton" name="center_alloc_submit">
        <property name="text">
         <string>Submit</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Garuda</family>
          <pointsize>12</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Paper Code</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="paper_code_list_2">
        <item>
         <property name="text">
          <string>121</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>122</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>123</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Manage</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>318</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionManage_Paper_Codes"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionManage_Paper_Codes">
   <property name="text">
    <string>Manage Paper Codes</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>paper_code_list</sender>
   <signal>textChanged(QString)</signal>
   <receiver>label_3</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>232</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>89</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>response_select</sender>
   <signal>clicked()</signal>
   <receiver>response_path</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>113</x>
     <y>200</y>
    </hint>
    <hint type="destinationlabel">
     <x>145</x>
     <y>200</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>key_select</sender>
   <signal>clicked()</signal>
   <receiver>key_path</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>305</y>
    </hint>
    <hint type="destinationlabel">
     <x>149</x>
     <y>305</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>student_info_select</sender>
   <signal>clicked()</signal>
   <receiver>student_info_path</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>197</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>center_infor_select</sender>
   <signal>clicked()</signal>
   <receiver>center_info_path</receiver>
   <slot>repaint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>130</x>
     <y>315</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>paper_code_list_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>label_10</receiver>
   <slot>setText(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>223</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>95</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
